src/main/java/seedu/address/logic/commands/DeleteCompanyCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.company.Company;
+
+
+/**
+ * Deletes a company identified using it's displayed index from the address book.
+ */
+public class DeleteCompanyCommand extends DeleteContactCommand {
```

Comment:
This is just a shortcut way to implement delete company so that the parser still works, will need to have delete contact and delete command inherit from a parent class

Reply:
Alright,  I think to keep track of this, you should add a // TODO comments in this file, and add this as an issue. 

After which this conversation can be resolved.